api/notd/block_processor.py

```diff
@@ -59,12 +67,45 @@ async def get_transfers_in_block(self, blockNumber: int) -> List[RetrievedTokenT
         block = await self.ethClient.get_block(blockNumber)
         blockHash = block['hash'].hex()
         blockDate = datetime.datetime.fromtimestamp(block['timestamp'])
-        events = await self.ethClient.get_log_entries(startBlockNumber=blockNumber, endBlockNumber=blockNumber, topics=[self.erc721TansferEventSignatureHash])
-        logging.info(f'Found {len(events)} events in block #{blockNumber}')
-        allTokenTransfers = []
-        for eventsChunk in list_util.generate_chunks(events, 5):
-            allTokenTransfers += await asyncio.gather(*[self._process_event(event=dict(event), blockNumber=blockNumber, blockHash=blockHash, blockDate=blockDate) for event in eventsChunk])
-        return [tokenTransfer for tokenTransfer in allTokenTransfers if tokenTransfer]
+        #erc721events = await self.ethClient.get_log_entries(startBlockNumber=blockNumber, endBlockNumber=blockNumber, topics=[self.erc721TansferEventSignatureHash])
+        #logging.info(f'Found {len(erc721events)} events in block #{blockNumber}')
+        erc1155events = await self.ethClient.get_log_entries(startBlockNumber=blockNumber, endBlockNumber=blockNumber, topics=[self.erc1155TansferEventSignatureHash])
+        print(erc1155events)
+        logging.info(f'Found {len(erc1155events)} erc1155events in block #{blockNumber}')
+
+        erc721TokenTransfers = []
+        erc1155TokenTransfers = []
+        #for erc721EventsChunk in list_util.generate_chunks(erc721events, 5):
```

Comment:
I would suggest you move all the below code into two functions like `_get_erc721_transfers(events=erc721events)` and `_get_erc1155_transfers(events=erc1155events)`

Reply:
(and then you can just comment out calling the function rather than having all this interwoven code below)